es6-modules/src/components/todo-form.js

```diff
@@ -0,0 +1,45 @@
+import createElement from '../lib/create-element';
```

Comment:
This works but it would be better to use:
```JAVASCRIPT
import { createElement } from '../lib';
```
This makes your code more maintainable and better for future updates. Plus it is a bit shorter.